guides/common/modules/proc_synchronizing-templates-with-a-local-directory-using-the-api.adoc

```diff
@@ -28,20 +28,24 @@ organizations:
 ----
 
 .Procedure
-. Create the directory where templates are stored and apply appropriate permissions and SELinux context:
+. In `/var/lib/foreman`, create a directory for storing templates:
 +
 [subs="+quotes"]
 ----
-# mkdir -p _/usr/share/templates_dir/_
-# chown foreman _/usr/share/templates_dir/_
-# chcon -t httpd_sys_rw_content_t _/usr/share/templates_dir/_ -R
+# mkdir /var/lib/foreman/_My_Templates_Dir_
```

Comment:
Is this even needed? I'd think the plugin creates this by itself (if it has permissions).

cc @theforeman/foreman_templates

Reply:
It depends on what values you provide in the export form.

`Repo` can be either full path or a "root" path. In this case it can be both `/var/lib/foreman/_My_Templates_Dir_` and `/var/lib/foreman`. In this case if `_My_Templates_Dir` doesn't exist, it won't be created.
`Dirname` can be omitted or specified. If specified then the plugin will create this directory in `Repo` root.

TL;DR:
`_My_Templates_Dir_` won't be created unless it's specified as `Dirname` in export form.